### PR TITLE
Update conda installation instructions

### PR DESCRIPTION
**Pull request by wrongkindofdoctor**
**Description**
Add a line advising users to install their own copies of Anaconda/Miniconda to start_install.rst

Associated issue # (replace this phrase and parentheses with the issue number)  

**Checklist:**
- [x] I have reviewed my own code to ensure that if follows the [POD development guidelines](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/dev_guidelines.html)
- [ ] The script are written in Python 3.6 or above (preferred; required if funded by a CPO grant), NCL, or R
- [ ] All of my scripts are in the diagnostics/ subdirectory, and include a main_driver script template html, and settings.jsonc
- [ ] The main_driver script header has all of the information in the POD documentation, excluding the "More about this diagnostic" section
- [ ] The POD directory and html template have the same short name as my POD
- [ ] The html template has a 1-paragraph synopsis of the POD and links to the main documentation
- [ ] If applicable, I've added a .yml file to src/conda, and my environment builds with conda_env_setup.sh 
- [ ] The POD scripts do not access the internet or networked resources
- [ ] I have commented the code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation in the POD's doc/ subdirectory
- [ ] I have created the directory input_data/obs_data/[pod short name]
- [ ] My code is portable; it uses [MDTF environment variables](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/ref_envvars.html), and does not contain hard-coded file or directory paths
- [ ] I have added information about the raw data files to the documentation
- [ ] I have provided the code to generate digested data files from raw data files
- [ ] Each digested data file generated by the script contains numerical data (no figures), and is 3 GB or less in size
- [x] The repository contains no extra test scripts or data files
- [x] My branch is up-to-date with the NOAA-GFDL develop branch, and all merge conflicts are resolved

### CODE REVIEW

**Review by jkrasting — APPROVED** [review]
Great addition clarifying the individual install.

**Review by tsjackson-noaa — CHANGES_REQUESTED** [review]
Hi @wrongkindofdoctor, I agree we need to go into more detail on our guidance with conda, but I think this wording is a bit misleading.

- For reasons I don't understand (and which are a pain in this case), the `conda`command  is implemented as a shell alias pointing to a hard-coded path to the "real" executable. If the user installs ana/miniconda over a site installation, that will redefine the alias and case everything provided through the site installation to be inaccessible, potentially including the user's other environments.

  This is why we caution users against multiple conda installations: we mention this a bit in [the docs](https://mdtf-diagnostics.readthedocs.io/en/latest/sphinx/start_install.html#installing-the-conda-package-manager), but it might be worth spelling out more explicitly.

- I don't think "avoiding version conflicts between conda and MDTF-diagnostics dependencies" is a good explanation. Questionable decisions like the one above aside, conda works like any other package manager -- there's no "conflict" between it and the packages it installs. The user's `$PATH` is prepended to when an environment is activated, so e.g. our NCL will be used instead of a system NCL that might be the wrong version. Finally, a package manager can install arbitrary packages newer than itself, unless breaking changes have been made.

  That's my guess as to what went wrong with the Hawaii group's installation: command syntax changed with conda 4.4, e.g. environments are activated with `conda activate` instead of `source activate`. This would break the existing scripts, but it'll be feasible to throw in a version check. Conda 4.4. came out in Dec 2017, so it's possible that's the newest version their site has.

**Review by jkrasting — APPROVED** [review]

**Comment by tsjackson-noaa** [comment]
Apologies for dragging this out, but I don't think the changes made in the most recent commits address the issues mentioned in my [earlier comment](https://github.com/NOAA-GFDL/MDTF-diagnostics/pull/208#pullrequestreview-657123052).

Asking users to re-create their pre-existing environments would be very time-consuming, and even more of an obstacle if the site installation's environments are used for collaboration with other users. 

I guess the issue is that I'm not seeing the motivation for telling users to replace the system installation with their own? As mentioned above, old versions of conda can install newer packages, and the only potential problem with old conda versions I'm aware of is the command syntax change for versions older than 4.4. We could test the site's version and wrap the commands in the installer script if it's important to support these.

**Comment by wrongkindofdoctor** [comment]
@tsjackson-noaa Okay. I omitted the explicit rec for self-installation. If the central install causes unforeseen issues, they can always use a self-managed version. Adding a conda version check to the installation script would probably be helpful overall.

File: doc/sphinx/start_install.rst [hunk 1]
```diff
@@ -110,7 +110,7 @@ Installing dependencies via the conda package manager
...
+For maximum portability and ease of installation, we recommend that all users manage dependencies through conda using the steps below, even if they have independent installations of the required languages. A complete installation of all dependencies will take roughly 5 Gb, less if you've already installed some of the dependencies through conda. The location of this installation can be changed with the ``--conda_root`` and ``--env_dir`` flags described below. Users may install their own copies of Anaconda/Miniconda on their machine, or use a centrally-installed version managed by their institution. Note that installing your own copy of Anaconda/Miniconda will re-define the default locations of the conda executable and environment directory if you have previously used a version of conda managed by your institution, so you will have to re-create any environments made using central conda installations.
```
Comment (jkrasting) [inline doc/sphinx/start_install.rst:113]:
```suggestion
For maximum portability and ease of installation, we recommend that all users manage dependencies through conda using the steps below, even if they have independent installations of the required languages. A complete installation of all dependencies will take roughly 5 Gb, less if you've already installed some of the dependencies through conda. The location of this installation can be changed with the ``--conda_root`` and ``--env_dir`` flags described below. Users may install their own copies of Anaconda/Miniconda on their machine, or use a centrally-installed version managed by their institution. Note that installing your own copy of Anaconda/Miniconda will re-define the default locations of the conda executable and environment directory defined in your `.bashrc` or `.cshrc` file if you have previously used a version of conda managed by your institution, so you will have to re-create any environments made using central conda installations. 
```

**Review by jkrasting — CHANGES_REQUESTED** [review]

**Review by jkrasting — APPROVED** [review]

**Review by tsjackson-noaa — APPROVED** [review]

**Comment by tsjackson-noaa** [comment]
Thanks @wrongkindofdoctor -- current wording makes the issue with personal vs. site installs more clear.

**Pull request merged**